skipper/runner.py

```diff
@@ -53,14 +52,12 @@ def _run_nested(fqdn_image, environment, command, interactive, name, net, volume
     for env in environment:
         docker_cmd += ['-e', env]
 
-    user = getpass.getuser()
-    user_id = os.getuid()
-    docker_cmd += ['-e', 'SKIPPER_USERNAME=%(user)s' % dict(user=user)]
-    docker_cmd += ['-e', 'SKIPPER_UID=%(user_id)s' % dict(user_id=user_id)]
     docker_cmd += ['-e', 'HOME=%(homedir)s' % dict(homedir=homedir)]
 
+    user_id = os.getuid()
     docker_gid = grp.getgrnam('docker').gr_gid
-    docker_cmd += ['-e', 'SKIPPER_DOCKER_GID=%(docker_gid)s' % dict(docker_gid=docker_gid)]
+
+    docker_cmd += ['-u', '%s:%s' % (user_id, docker_gid)]
```

Comment:
I think you should use:
--user=user_id:user_group_id --group-add docker_group_id